src/definition/externalComponent/IWebHooks.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * This interface provides webhooks to the game. You can set
```

Comment:
Not sure I understand why we will have a second webhook/endpoint in the codebase. As a developer, how will I know when to use what? Why can't i use the other endpoint system already in the apps for these external components? Also, this documentation still refers to games.

Reply:
The idea is to replace this with event listener interfaces, similar to `IPreMessageSent`

Reply:
Yes, that sounds like a much better solution. 👍